src/server/analytic_services/prometheus_reporting.js

```diff
@@ -296,7 +296,15 @@ const METRIC_RECORDS = Object.freeze([{
         name: get_metric_name('system_links'),
         help: 'System Links',
         labelNames: ['resources', 'buckets', 'dashboard'],
-    }
+    },
+}, {
```

Comment:
looks like weird indentation ?

Reply:
what's weird to you? I think all the others look the same to me :)